lib/optimizely/config/datafile_project_config.rb

```diff
@@ -194,6 +198,26 @@ def experiment_running?(experiment)
       RUNNING_EXPERIMENT_STATUS.include?(experiment['status'])
     end
 
+    def get_sdk_key()
```

Comment:
why create these getters when you have already declared read only attributes and using those directly?